templates/apps/sumo-aws-apps/firewall-manager-org/firewall-manager-org-sumo-logsource.yaml

```diff
@@ -122,17 +114,14 @@ Resources:
           - id: W35
             reason: S3 access logging not enabled
           - id: W51
-            reason: S3 bucket should likely have a bucket policy            
+            reason: S3 bucket should likely have a bucket policy    
+          - id: W41
+            reason: S3 Bucket should have encryption option set                     
```

Comment:
We should set encryption option I think last time AWS said that they want S3 bucket encryption option.

Reply:
ok will add BucketEncryption

Reply:
lets remove W41